tests/Agent/IntegrationTests/IntegrationTestHelpers/RemoteServiceFixtures/ConsoleDynamicMethodFixture.cs

```diff
@@ -150,7 +157,31 @@ public ConsoleDynamicMethodFixtureCoreLatest()
     /// <summary>
     /// Use this fixture for High Security Mode tests
     /// </summary>
-    public class ConsoleDynamicMethodFixtureCoreLatestHSM : ConsoleDynamicMethodFixtureCore70
+    public class ConsoleDynamicMethodFixtureCore70HSM : ConsoleDynamicMethodFixtureCore70
+    {
+        public override string TestSettingCategory { get { return "HSM"; } }
+        public ConsoleDynamicMethodFixtureCore70HSM()
+        {
+        }
+
+    }
+
+    /// <summary>
+    /// Use this fixture for Configurable Security Policy tests
+    /// </summary>
+    public class ConsoleDynamicMethodFixtureCore70CSP : ConsoleDynamicMethodFixtureCore70
+    {
+        public override string TestSettingCategory { get { return "CSP"; } }
+        public ConsoleDynamicMethodFixtureCore70CSP()
+        {
+        }
+
+    }
```

Comment:
Do we really need to run this test on both .net 7 and .net 8? I don't think that is is testing anything specific to the frameworks.

Reply:
I think we could get away with not running it in both.